modules/modal/react/lib/Modal.tsx

```diff
@@ -12,7 +12,6 @@ export enum ModalWidth {
 export interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
   open: boolean;
   padding: PopupPadding;
-  transformOrigin: TransformOrigin;
```

Comment:
This will probably conflict with #184

Reply:
Good call. We can wait until that one is in.

Reply:
Rebased and removed my notes about `transformOrigin` 👍